ChocolateyPackage/textanalysistoolnet.portable.nuspec

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>textanalysistoolnet.portable</id>
+    <version>2018.11.20</version>
+    <owners>bcurran3</owners>
```

Comment:
Who is this?

Reply:
Missed to update this from the sample file.